source/retryable-writes/tests/README.rst

```diff
@@ -175,6 +175,26 @@ Each YAML file has the following keys:
       - ``data``: The data that should exist in the collection after the
         operation has been run.
 
+Split Batch Tests
+=================
+
+The YAML tests specify bulk write operations that are split by command type
+(e.g. sequence of insert, update, and delete commands). Multi-statement write
+operations may also be split due to ``maxWriteBatchSize`` or
+``maxBsonObjectSize``. For instance, an insertMany operation with three 9 MB
+documents would be split into three insert commands in order to respect the 16
+MB ``maxBsonObjectSize`` limit. Drivers should manually test that operations
+split due to ``maxWriteBatchSize`` and ``maxBsonObjectSize`` can still be
+retried.
+
+The ``onPrimaryTransactionalWrite`` fail point's ``skip`` option may be used to
+control when the fail point triggers. When testing a split insert due to
+``maxBsonObjectSize``, a single skip will allow the fail point to trigger on the
+second insert command. When testing a split update due to ``maxWriteBatchSize``,
+a skip of ``maxWriteBatchSize`` would allow the fail point to trigger on the
```

Comment:
"a single skip will also allow" 

A single insert command -- even with multiple documents --  only counts towards the skip once because the entire batch is committed together. I'm not sure if the same is true for updates and deletes.